helm/opendistro-es/templates/elasticsearch/es-master-sts.yaml

```diff
@@ -188,37 +188,37 @@ spec:
           subPath: admin-root-ca.pem
         {{- end }}
         {{- if .Values.elasticsearch.securityConfig.enabled }}
-        {{- if .Values.elasticsearch.securityConfig.actionGroupsSecret }}
+        {{- if .Values.elasticsearch.securityConfig.actionGroups }}
         - mountPath: {{ .Values.elasticsearch.securityConfig.path }}/action_groups.yml
-          name: action-groups
+          name: config
           subPath: action_groups.yml
         {{- end }}
-        {{- if .Values.elasticsearch.securityConfig.configSecret }}
+        {{- if .Values.elasticsearch.securityConfig.config }}
         - mountPath: {{ .Values.elasticsearch.securityConfig.path }}/config.yml
-          name: security-config
+          name: config
           subPath: config.yml
         {{- end }}
-        {{- if .Values.elasticsearch.securityConfig.internalUsersSecret }}
+        {{- if .Values.elasticsearch.securityConfig.internalUsers }}
         - mountPath: {{ .Values.elasticsearch.securityConfig.path }}/internal_users.yml
-          name: internal-users-config
+          name: config
           subPath: internal_users.yml
         {{- end }}
-        {{- if .Values.elasticsearch.securityConfig.rolesSecret }}
+        {{- if .Values.elasticsearch.securityConfig.roles }}
         - mountPath: {{ .Values.elasticsearch.securityConfig.path }}/roles.yml
-          name: roles
+          name: config
           subPath: roles.yml
         {{- end }}
-        {{- if .Values.elasticsearch.securityConfig.rolesMappingSecret }}
+        {{- if .Values.elasticsearch.securityConfig.rolesMapping }}
         - mountPath: {{ .Values.elasticsearch.securityConfig.path }}/roles_mapping.yml
-          name: role-mapping
+          name: config
           subPath: roles_mapping.yml
         {{- end }}
-        {{- if .Values.elasticsearch.securityConfig.tenantsSecret }}
+        {{- if .Values.elasticsearch.securityConfig.tenants }}
         - mountPath: {{ .Values.elasticsearch.securityConfig.path }}/tenants.yml
-          name: tenants
+          name: config
           subPath: tenants.yml
         {{- end }}
-      {{- end }}
+        {{- end }}
```

Comment:
Have you tried mounting the security secret in the security config path directly without specificing each individual subPath?
```
mountPath: {{ .Values.elasticsearch.securityConfig.path }}
          name: securtiy-config
```

Reply:
Oh, that was what I did... It is already pushed to CK8S (Check lines 191--193 https://github.com/elastisys/ck8s/blob/opendistr-es/helmfile/charts/opendistro-es/templates/elasticsearch/es-master-sts.yaml) I thought I pushed it here as well. I will push it here as well

Reply:
Pushed now!